server/api/utils.js

```diff
@@ -34,6 +35,27 @@ router.post("utils.gc", async (ctx) => {
 
   log(`Destroyed ${countDeletedDocument} documents`);
 
+  log(`Expiring all the collection export older than 30 days…`);
```

Comment:
This is great, sorry to be a pain but we'll need a test for this addition – wouldn't want current exports being deleted.

Reply:
testing is good for saving headaches later 💪🏻